envs/dev/templates/team-geordi.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: team-geordi
+  namespace: argocd
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  destination:
+    namespace: yelb
+    server: {{ .Values.spec.destination.server }}
+  source:
+    repoURL: {{ .Values.spec.source.repoURL }}
```

Comment:
Why not just pass the URL (or hard code it) to the original gitops repo?

Reply:
Yes im following this now in the updated PR.